about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./assets/style/index.css">
+	<title>Tourism Website</title>
+</head>
+
+<body>
+	<header>
+
+		<nav class="menu">
+			<div class="burger">
+				<div class="line1"></div>
+				<div class="line2"></div>
+				<div class="line3"></div>
+			</div>
+		</nav>
+
+		<ul class="nav-ul-first">
+			<li class="navbar-first"><a class="nav-link-first" href="#">English</a></li>
+			<li class="navbar-first"><a class="nav-link-first" href="#">My Page</a></li>
+			<li class="navbar-first"><a class="nav-link-first" href="#">Logout</a></li>
+		</ul>
+		<ul class="nav-ul">
+			<li><a href="./index.html">
+					<img class="logo-img" src="./assets/images/logo.svg" alt=""></a></li>
+			<li class="navbar"><a class="nav-link" href="./index.html">Home</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_2_about">Events</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_3_about">Tours</a></li>
+			<li class="navbar"><a class="nav-link" href="footer">Contact</a></li>
+		</ul>
+	</header>
+	<section class="section_1_about" id="section_1_about">
+		<h2 class="main-text-about">
+			Your Perfect Tour in Uzbekistan Starts Here!
+		</h2>
+		<p class="text-about">
+			Lorem Ipsum Travel has passed a long way since its early beginnings in 1988. The company was founded by
+			two ordinary guys passionate about tourism. At the moment, the company serves more than 15,000 travellers
+			across Central Asia each year.
+
+			Our professional team has been lucky to travel to a number of amazing countries across all continents and
+			can proudly boast many years of practical expertise in the travel industry. They are committed to
+			excellence in providing clients with high-quality travel and the best possible traveling experience at
+			reasonable prices. Due to the time spent abroad, they clearly understand all the demands of a modern
+			traveler. Our specialists are always friendly and pay individual attention to each client. Expert guides
+			will happily share with you all of their favourite spots
+		</p>
+	</section>
+
```

Comment:
- [ ] As requested by the previous code reviewer, let's adjust the **line height** as well as  **font sizes** to match the given [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)